docs/examples/managed/GoogleDemo.ipynb

```diff
@@ -37,9 +37,71 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "### Authentication: Get OAuth Credentials\n",
+    "### Google Authentication Overview\n",
     "\n",
-    "The Google Semantic Retriever API lets you perform semantic search on your own data. Since it's **your data**, this needs stricter access controls than API Keys. Please follow [OAuth Quickstart](https://developers.generativeai.google/tutorials/oauth_quickstart) to setup OAuth authentication. Below are overview of steps from the documentation that are required.\n",
+    "The Google Semantic Retriever API lets you perform semantic search on your own data. Since it's **your data**, this needs stricter access controls than API Keys. Authenticate with OAuth through service accounts or through your user credentials.\n",
+    "\n",
+    "**Note**: At this time, the Google Generative AI Semantic Retriever API is [only available in certain regions](https://ai.google.dev/available_regions)."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### Authentication (Option 1): OAuth using service accounts\n",
+    "\n",
+    "To OAuth using service accounts, follow the steps below:\n",
+    "\n",
+    "1. Enable the `Generative Language API`: [Documentation](https://developers.generativeai.google/tutorials/oauth_quickstart#1_enable_the_api)\n",
+    "\n",
+    "1. Create the Service Account by following the [documentation](https://developers.google.com/identity/protocols/oauth2/service-account#creatinganaccount).\n",
+    "\n",
+    " * After creating the service account, generate a service account key.\n",
+    "\n",
+    "1. Upload your service account file by using the file icon on the left sidebar, then the upload icon, as shown in the screenshot below.\n",
+    "\n",
+    "<img width=400 src=\"https://developers.generativeai.google/tutorials/images/colab_upload.png\">"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%pip install google-auth\n",
+    "%pip install google-auth-oauthlib"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from google.oauth2 import service_account\n",
+    "from llama_index.indices.managed.google.generativeai import (\n",
+    "    GoogleIndex,\n",
+    "    set_auth_credentials,\n",
```

Comment:
Should it be set_google_auth_credentials instead?

Reply:
ah yeah.. fixed!